assets/js/modules/analytics/datastore/index.js

```diff
@@ -50,6 +51,7 @@ const store = Data.combineStores(
 	properties,
 	profiles,
 	settings,
+	createSnapshotStore( STORE_NAME ),
```

Comment:
We could add this to `commonStore` now if you'd like?

Reply:
We could, but I just noticed we actually don't use `commonStore` at the moment in the `modules/*` stores, which we probably should. Alternatively we could start by adding `createSnapshotStore` to `createModuleStore` so that all modules have it for now?